### PR TITLE
Add custom control option for RuleGroup component

### PR DESCRIPTION
**Pull request by rbalaine**
This pull request allow to customize the RuleGroup component. As Rule component is a part of it, it's now accessible when using this module. Furthermore, I fixed an issue with copy-webpack-plugin (caused by the recent package upgrade).

### CODE REVIEW

**Comment by jakeboone02** [comment]
Thanks, @rbalaine!  Does this resolve #133?

File: index.d.ts [hunk 1]
```diff
@@ -62,6 +62,11 @@ interface NotToggleCustomControlProps extends CommonCustomControlProps {
   handleOnChange?(checked: boolean): void;
 }
 
+interface RuleGroupCustomControlProps extends CommonCustomControlProps {
+  checked: boolean
+  handleOnChange?(checked: boolean): void;
+}
+
```
Comment (jakeboone02) [inline index.d.ts]: These are incorrect.  They should match the props for `<RuleGroup />`.  Something like this:

```ts
interface RuleGroupCustomControlProps {
  id?: string;
  parentId?: string;
  combinator?: string;
  rules?: [];
  translations?: {};
  schema?: {};
  not?: boolean;
}
```
We may need to specify the `schema` attributes.  For `translations`, we can reuse the definition from below under `interface QueryBuilderProps {...}`.

File: src/index.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
 import QueryBuilder from './QueryBuilder';
 export default QueryBuilder;
 export { formatQuery } from './utils';
+import Rule from './Rule';
+export { Rule };
```
Comment (jakeboone02) [inline src/index.js]: Here you can do this in one line:
```js
export { Rule } from './Rule';
```

**Review by jakeboone02 — CHANGES_REQUESTED** [review]
@rbalaine - please see review comments

**Comment by rbalaine** [comment]
@jakeboone02 Thanks for your comments, I'm fixing that. It doesn't fix #133, only RuleGroup is now customizable. But I saw that artenator did it on his fork, he could make a Pull Request for it.

**Review by jakeboone02 — APPROVED** [review]

**Pull request merged**